manager/rbac_manager.py

```diff
@@ -109,13 +109,17 @@ def _find_host_groups(role_list) -> List[List[Dict[str, str]]]:
             if role.get("permission") not in {"inventory:hosts:read", "inventory:hosts:*"}:
                 continue
             no_permission = False
-            for rscdef in role.get('resourceDefinitions', []):
+            definitions = role.get('resourceDefinitions', [])
+            # When resourceDefinitions is [], put None to res to include ungroupped systems
+            if not definitions:
+                res.append(None)
+            for rscdef in definitions:
                 attrfilter = rscdef.get('attributeFilter', {})
                 if attrfilter.get('key') != 'group.id':
                     continue
                 if attrfilter.get('value') == []:
                     # if there are only [] values /no access to hosts/, `set(res)` will be == {""}
-                    res.extend([""])
+                    res.append("")
                 res.extend(attrfilter.get('value', []))
```

Comment:
this line can cause `Unhashable type list` exception in later `set(res) == {""}` if `value` is missing in rbac response, e.g.
```
permisions: something:*:something
resourceDefinition: [
    attributeFilter: {
        key: "group.id"
    }
]
```
let's hope they are sending always correct schema